diff --git a/pyproject.toml b/pyproject.toml
index 19b81705..3a69e4d4 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -50,11 +50,15 @@ repository = "https://github.com/dbekaert/RAiDER"
 "generateGACOSVRT.py"   = "RAiDER.models.generateGACOSVRT:main"
 
 [tool.setuptools]
+include-package-data = true
 zip-safe = false
 
 [tool.setuptools.packages.find]
 where = ["tools"]
 
+[tool.setuptools.package-data]
+"*" = ["*.yml", "*.yaml"]
+
 [tool.isort]
 known_first_party = "RAiDER"
 multi_line_output = 5
