From ea35fb44184127ef968f724723ef4680d023806b Mon Sep 17 00:00:00 2001
From: Joseph H Kennedy <me@jhkennedy.org>
Date: Tue, 10 Jan 2023 21:16:45 -0900
Subject: [PATCH 1/2] Add a maniftest.in for conda build

---
 MANIFEST.in | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 MANIFEST.in

diff --git a/MANIFEST.in b/MANIFEST.in
new file mode 100644
index 00000000..98740492
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1,8 @@
+include LICENSE
+include LICENSE2.md
+include README.md
+include CHANGELOG.md
+
+graft tools
+
+global-exclude *.py[cod] __pycache__ *.so

From b2e36ce90485b9275fd6cceda2ece9782113361c Mon Sep 17 00:00:00 2001
From: Joseph H Kennedy <me@jhkennedy.org>
Date: Wed, 11 Jan 2023 14:30:59 -0900
Subject: [PATCH 2/2] update manifest

---
 MANIFEST.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MANIFEST.in b/MANIFEST.in
index 98740492..f15fc4e4 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -3,6 +3,6 @@ include LICENSE2.md
 include README.md
 include CHANGELOG.md
 
-graft tools
+graft tools/RAiDER
 
 global-exclude *.py[cod] __pycache__ *.so
