diff --git a/pyproject.toml b/pyproject.toml
index f01a75a1..d5815cdb 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -58,7 +58,7 @@ zip-safe = false
 where = ["tools"]
 
 [tool.setuptools.package-data]
-"*" = ["*.yml", "*.yaml"]
+"*" = ["*.yml", "*.yaml", "*.zip"]
 
 [tool.isort]
 known_first_party = "RAiDER"
diff --git a/setup.py b/setup.py
index 9f163fc7..a8a76bc1 100755
--- a/setup.py
+++ b/setup.py
@@ -44,5 +44,4 @@
 setup(
     ext_modules=cython_extensions + pybind_extensions,
     cmdclass={"build_ext": build_ext},
-    package_data={'tools': ['RAiDER/models/*.zip']}
 )
